src/database/index.ts

```diff
@@ -1,11 +1,19 @@
-import createConnection from "./createConnection";
+import getDataSource from "./createConnection";
 
 const initializeDatabase = async (): Promise<void> => {
   try {
-    await createConnection();
+    await getDataSource().initialize();
   } catch (error) {
     console.log(error);
   }
 };
 
-export { initializeDatabase };
+const destroyDatabaseConnection = async (): Promise<void> => {
+  try {
+    await getDataSource().destroy();
+  } catch (error) {
+    console.log(error);
+  }
+};
+
+export { initializeDatabase, destroyDatabaseConnection };
```

Comment:
It's pretty much the same pattern as we follow in server.ts. We segregate instance of target and initialize/destroy with individual functions.